app/Controllers/WordController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controllers;
+
+use App\Entities\Response;
+use App\Enumerators\HttpHeaderStatus;
+use App\Exception\NotFoundException;
+use App\Exception\UnprocessableEntityException;
+use App\Repositories\WordRepository;
+
+readonly class WordController implements ControllerInterface
+{
+    public function __construct(
+        private WordRepository $wordRepository,
+    ) {
+    }
+
+    public function getAll(): Response
+    {
+        $result = $this->wordRepository->getAllWords();
+
+        return new Response(HttpHeaderStatus::OK->value, json_encode($result));
+    }
+
+    public function getById(int $id): Response
+    {
+        $wordEntity = $this->wordRepository->getWordById($id);
+        if($wordEntity === null) {
+            throw new NotFoundException('Not Found');
```

Comment:
__Optional__

You could add that word was not found 😄 and maybe include id.

"Word with id '{$id}' was not found"

Reply:
At this moment, the exception message gets built into the header as well. So it looks odd:

<img width="651" alt="image" src="https://github.com/user-attachments/assets/1bf08a9e-760e-478d-aeea-135337d3039d">

Reply:
But it'll get changed later!